picard/formats/id3.py

```diff
@@ -229,6 +230,8 @@ class ID3File(File):
         'MVIN': re.compile(r'^(?P<movementnumber>\d+)(?:/(?P<movementtotal>\d+))?$')
     }
 
+    __lrc_re_parse = re.compile(r'(\[\d\d:\d\d\.\d\d])')
```

Comment:
Not strictly needed, but:

```suggestion
    __lrc_re_parse = re.compile(r'(\[\d\d:\d\d\.\d\d\])')
```